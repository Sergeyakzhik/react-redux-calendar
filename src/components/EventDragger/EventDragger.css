.table-month .dragger {
  position: absolute;
  right: 0;
  width: 100%;
  height: 100%;
  top: 0;
  cursor: grab;
  z-index: 199;
  background: #4e3fa080;
}

.table-month .dragger:active {
  position: absolute;
  left: -178px;
  width: 330%;
  height: 1300%;
  top: -145px;
  cursor: -webkit-grab;
  cursor: grabbing;
}

.table-week .dragger {
  position: absolute;
  right: 0;
  width: 100%;
  height: 100%;
  top: 0;
  cursor: grab;
  z-index: 199;
  background: #4e3fa080;
}

.table-week .dragger:active {
  position: absolute;
  left: -178px;
  width: 330%;
  top: -145px;
  cursor: -webkit-grab;
  cursor: grabbing;
}
