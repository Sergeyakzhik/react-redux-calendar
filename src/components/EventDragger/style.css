.table-month .dragger {
  position: absolute;
  right: 0;
  width: 100%;
  height: 100%;
  top: 0;
  cursor: grab;
  z-index: 199;
}

.table-month .dragger:active {
  position: absolute;
  left: -178px;
  width: 330%;
  height: 1300%;
  top: -145px;
  cursor: -webkit-grab;
  cursor: grabbing;
}

.table-week .dragger, .table-day .dragger {
  position: absolute;
  right: 0;
  width: 100%;
  height: 100%;
  top: 0;
  cursor: grab;
  z-index: 199;
}

.table-week .dragger:active, .table-day .dragger:active {
  position: absolute;
  left: -178px;
  width: 330%;
  top: -145px;
  cursor: -webkit-grab;
  cursor: grabbing;
}

.table-day .dragger:active {
  left: -178px;
  width: 330%;
  top: -145px;
}
