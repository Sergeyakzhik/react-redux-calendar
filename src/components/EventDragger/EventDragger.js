import React from 'react';
import './EventDragger.css';

let EventDragger = props => (
  <div
    className="dragger"
    targetKey={props.targetKey}
    onMouseDown={props.onMouseDown}
    onMouseUp={props.onMouseUp}
    onClick={e => e.stopPropagation()}
    style={props.style}
  >
  </div>
);

export default EventDragger;
