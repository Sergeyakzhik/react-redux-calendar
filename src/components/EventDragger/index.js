import React from 'react';
import './style.css';

let EventDragger = props => (
  <div
    className="dragger"
    targetKey={props.targetKey}
    onMouseDown={props.onMouseDown}
    onMouseUp={props.onMouseUp}
    onMouseLeave={props.onMouseLeave}
    onClick={e => e.stopPropagation()}
    style={props.style}
  >
  </div>
);

export default EventDragger;
