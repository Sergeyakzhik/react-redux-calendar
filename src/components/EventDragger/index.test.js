import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import EventDragger from './index';

Enzyme.configure({ adapter: new Adapter() });

describe('EventDragger', () => {
  const props = {
    style: {},
    targetKey: '',
    onMouseUp: () => {},
    onMouseDown: () => {},
    onMouseLeave: () => {},
    onClick: () => {},
  };
  const component = shallow(<EventDragger {...props} />);

  it('renders EventDragger component', () => {
    expect(component).toMatchSnapshot();
  });
});
