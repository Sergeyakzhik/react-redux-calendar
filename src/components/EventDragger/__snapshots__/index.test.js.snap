// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EventDragger render EventDragger component 1`] = `
<div
  className="dragger"
  onClick={[Function]}
  onMouseDown={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  style={Object {}}
  targetKey=""
/>
`;
